platform/cog-platform-drm.c

```diff
@@ -88,13 +101,22 @@ static struct {
     struct wpe_input_touch_event_raw touch_points[10];
     enum wpe_input_touch_event_type last_touch_type;
     int last_touch_id;
+
+    struct cursor cursor;
```

Comment:
This can be a standalone globa. DRM-related object ownership doesn't fit into `input_data`.